### PR TITLE
Update to data-migration.md

### PR DESCRIPTION
**Pull request by jsindt**
One note: it might be worth removing the `scp` section given that we're pushing for `rsync` -- happy for comments.

### CODE REVIEW

**Review by aturner-epcc — CHANGES_REQUESTED** [review]
Looks fine, just a couple of minor comments

File: docs/archer-migration/data-migration.md [hunk 1]
```diff
@@ -2,78 +2,56 @@
...
-   in SAFE.
-   
-!!! tip
-    Remember that you will need to use both a key and your password to
-    transfer data to ARCHER2.
+!!! note
+    This section assumes that you have an active ARCHER and ARCHER2 account, and that you 
+    have successfully logged in to both accounts.
+
+First, login to the ARCHER(1) data transfer node (making sure to change
```
[the remark this answers is not in the document]
Comment (jsindt) [reply docs/archer-migration/data-migration.md]: Good spot -- fixed.

File: docs/archer-migration/data-migration.md [hunk 2]
```diff
@@ -2,78 +2,56 @@
...
-2. Add the public part of the SSH key you create to your ARCHER2 account 
-   in SAFE.
-   
-!!! tip
-    Remember that you will need to use both a key and your password to
-    transfer data to ARCHER2.
+!!! note
+    This section assumes that you have an active ARCHER and ARCHER2 account, and that you 
+    have successfully logged in to both accounts.
+
```
[the remark this answers is not in the document]
Comment (jsindt) [reply docs/archer-migration/data-migration.md:13]: Tip added.

**Review by jsindt — COMMENTED** [review]
Thanks for reviewing.

**Review by aturner-epcc — APPROVED** [review]
Thanks @jsindt Merging now

**Pull request merged**